src/components/Agreement/InfoField.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
Just wondering, maybe we can move this component up so it can be used by both apps. What do you think? I have a very similar one on disputable voting app

Reply:
Good idea! it's quite a common need.